src/templates/items/asi.hbs

```diff
@@ -44,8 +44,8 @@
                         <h3 class="bubble-header">{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.Instruction"}}</h3>
 
                         <div class="form-group stacked bubble-info">
-                            {{#each config.abilities as |ability name|}}
-                            <label class="checkbox" data-tooltip="{{lookup (lookup ../config.abilities name) "tooltip"}}">
+                            {{#each (sfrpg "abilities") as |ability name|}}
+                            <label class="checkbox">
```

Comment:
Apparently `CONFIG.SFRPG.abilities[name]` is a string, not an object, so it doesn't have a `tooltip` property. But `lookup` will quietly handle that, so I guess this was a latent problem that replacing `lookup` with the config helper dragged up.

This is the only problem of it's kind that I've noticed, and I've now checked creating every type of item. Let me know there are any other problems.